melosys-eessi-app/src/main/java/no/nav/melosys/eessi/closebuc/LukkBucAarsakMapper.java

```diff
@@ -13,25 +13,27 @@ private LukkBucAarsakMapper() {}
     private static final String TRETTI_DAGER_SIDEN_A008 = "lovvalg_30_dager_siden_melding_om_relevant_informasjon";
     private static final String TRETTI_DAGER_SIDEN_MELDING_OM_UTSTASJONERING = "lovvalg_30_dager_siden_melding_om_utstasjonering";
     private static final String TRETTI_DAGER_SIDEN_SVAR_ANMODNING_MER_INFO = "gjeldende_lovgivning_30_dager_siden_svar_på_anmodning_om_mer_informasjon";
+    private static final String ENIGHET_ANMODNING_UNNTAK = "gjeldende_lovgivning_det_ble_oppnådd_enighet_om_anmodningen_om_unntak";
```

Comment:
Nevner ikke etter 2 mnd? Jeg spør fordi andre begrunnelser nevner dager.

Reply:
Ja, er forskjellige regler for alle BUCer. Ikke 100% sikker på hva (alle) reglene er for LA_BUC_01, men den mest vanlige er om vi har sendt A001, og har mottatt enten A002 eller A011.

Vi legger på en tilleggsregel på at vi ikke skal ha mottatt noe på 2 mnd. Dette er i hovedsak for å lukke gamle BUCer. Vi skal ellers primært lukke LA_BUC_01 som følge av vedtak i melosys-api, det blir neste PR 🙂